homeassistant/components/fully_kiosk/services.py

```diff
@@ -81,3 +97,19 @@ async def async_start_app(call: ServiceCall) -> None:
                 )
             ),
         )
+
+    hass.services.async_register(
+        DOMAIN,
+        SERVICE_SET_CONFIG,
+        async_set_config,
+        schema=vol.Schema(
+            vol.All(
+                {
+                    vol.Required(ATTR_DEVICE_ID): cv.ensure_list,
+                    vol.Required(ATTR_CONFIG_TYPE): vol.In(["string", "bool"]),
+                    vol.Required(ATTR_KEY): cv.string,
+                    vol.Required(ATTR_VALUE): vol.Any(cv.string, cv.boolean),
```

Comment:
This won't work and will translate `True` and `False` to string. You just want to actually match against the types.
```suggestion
                    vol.Required(ATTR_VALUE): vol.Any(str, bool),
```

Reply:
The FKB API is weird, it has separate functions for setting a config value depending on if its (what they determine to be) a string or a bool value, but it expects "true" and "false" as strings for bool values 🙃